README.md

```diff
@@ -2,55 +2,256 @@
-| local/ip             | A key to the local dictionary defining the IP address of the local participant&#39;s public network interface. | str | no | - |
-| local/hostname       | A key to the local dictionary defining the DNS hostname of the local participant&#39;s public network interface | str | no | - |
-| local/subnet         | A key to the local dictionary defining the subnet of the local participant&#39;s public network interface. | str | no | - |
-| local/rsasigkey      | A key to the local dictionary defining the public key for RSA signature authentication.           | str         | no       | -                       |
-| local/cert           | A key to the local dictionary defining the certificate nickname of your certificate in the NSS database | str | no | - |
-| remote               | Defines information specific to the remote participant&#39;s public network interface. Only defined when the use case requires a single-host setup. | dict | no | - |
-| remote/ip            | A key to the remote dictionary defining the IP address of the remote participant&#39;s public network interface. | str | no | - |
-| remote/hostname      | A key to the remote dictionary defining the DNS hostname of the remote participant&#39;s public network interface | str | no | - |
-| remote/subnet        | A key to the remote dictionary defining the subnet of the remote participant&#39;s public network interface. | str | no | - |
-| remote/rsasigkey     | A key to the remote dictionary defining the public key for RSA signature authentication           | str         | no       | -                       |
-| remote/cert          | A key to the remote dictionary defining the certificate nickname of your certificate in the NSS database | str | no | - |
+| name                 | An unique, arbitrary name used to identify the connection.                                        | str         | no       | See [name](#name)       |
+| hosts                | A list of peers. A vpn tunnel will be constructed between each pair of hosts in this list. <br /> See [hosts](#hosts). | list | no | -               |
 | ike                  | Defines information specific to setting up the IKE protocol.                                      | dict        | no       | -                       |
-| ike/alg              | A key to the ike dictionary defining which encryption/authentication algorithm to use for the connection. | str | no | vpn\_enc\_alg                 |
-| ike/lifetime         | A key to the ike dictionary defining how long the keying channel of a connection should last before being renegotiated | str | no | vpn\_lifetime    |
+| ike/alg              | A key to the ike dictionary defining which encryption/authentication algorithm to use <br /> for the connection. | str | no | vpn\_enc\_alg          |
```

Comment:
I would suggest using `ike.alg` instead of `ike/alg` to indicate that `alg` is a sub-key of the `ike` `dict` - this will use the same syntax as accessing a member as you would use in Ansible or python.